### PR TITLE
fixed the remote publication with empty test

### PR DESCRIPTION
**Pull request by mike-ionut-mihai-sandu-tao**
fixed the empty test(no items issue) 
https://oat-sa.atlassian.net/browse/NEX-621
it depends on : 
https://github.com/oat-sa/extension-tao-test/pull/312

### CODE REVIEW

File: model/delivery/factory/RemoteDeliveryFactory.php [hunk 1]
```diff
@@ -71,6 +72,16 @@ public function createPublication(
             $test, $deliveryClass->getLabel(), $deliverTenant->getLabel()
         ));
 
+        /** @var TestsService $testService */
+        $testService = $this->getTestService();
```
Comment (boajer) [inline model/delivery/factory/RemoteDeliveryFactory.php]: Can we somehow avoid code duplication? Fragment above ( https://github.com/oat-sa/extension-tao-deliver-connect/pull/42/files#diff-8ec4ee2cb8b864d67d5aa7da5a176429R231 ) looks the same ?

Comment (mike-ionut-mihai-sandu-tao) [reply model/delivery/factory/RemoteDeliveryFactory.php]: done
https://github.com/oat-sa/extension-tao-test/pull/312

File: controller/PublishDelivery.php [hunk 2]
```diff
@@ -227,6 +228,14 @@ protected function validateDeliverConnectForm(tao_helpers_form_Form $remoteDeliv
         }
 
         $test = $this->getResource($remoteDeliveryForm->getValue('test'));
+        /** @var TestsService $testService */
+        $testService = $this->getTestService();
+        if (!$testService->hasItems($test)) {
```
Comment (boajer) [inline controller/PublishDelivery.php]: This depends on functionality introduced in taoTest version `13.4.3`, it should be reflected at `requires` section of manifest

**Comment by mike-ionut-mihai-sandu-tao** [comment]
checking now the possibility of the LTI publication

**Comment by mike-ionut-mihai-sandu-tao** [comment]
i have tested the LTI -> provider / consumer flow by publishing an local test and getting the LTI link that i used to create a new publication using the LTI tab and running it as guest

**Pull request merged**